convert/convert.go

```diff
@@ -53,11 +53,13 @@ func (Parameters) writeFile(ws io.WriteSeeker, kv llm.KV, ts []llm.Tensor) error
 	return llm.WriteGGUF(ws, kv, ts)
 }
 
+type NameFunc func(string) string
+
 type Converter interface {
 	// KV maps parameters to LLM key-values
 	KV(*Tokenizer) llm.KV
 	// Tensors maps input tensors to LLM tensors. Model specific modifications can be done here.
-	Tensors([]Tensor) []llm.Tensor
+	Tensors([]Tensor, NameFunc) []llm.Tensor
```

Comment:
Can we get away without this parameter to avoid making the interface more complex (and harder to inherit)

Reply:
I don't see how without adding more indirection. The problem is gemma implements Tensors for both gemma and gemma2 but gemma2 has additional renames. Ideally the gemma2 renames does not bleed into the gemma implementation. Same thing with mixtral and llama